cmd/common.go

```diff
@@ -242,7 +242,10 @@ func loadConfig(nodetype check.NodeType) string {
 func isMaster() bool {
 	glog.V(2).Info("Checking if the current node is running master components")
 	masterConf := viper.Sub(string(check.MASTER))
-	components, err := getBinaries(masterConf)
+	if masterConf == nil {
```

Comment:
We should add a log statement indicating why isMaster is returning `false` in this situation. It'll help with troubleshooting later on. I think glog.V(2).Info(...) should be fine.

Reply:
Added.